### PR TITLE
prevent sql inject

### PR DESCRIPTION
**Pull request by lesleykhuu**
prevent sql inject and attempted to fix view and controller logic

### CODE REVIEW

**Comment by ontraportgithub** [comment]
Looks good to me.

**Comment by lesleykhuu** [comment]
also changed index and loginerror to .html files because php was not used in either

File: crmlogin/contactcreate.php [hunk 1]
```diff
@@ -1,12 +1,37 @@
...
+				$message = "This email is already used by an existing contact<br><br>";
+			else{
+				$sql = "INSERT INTO contactlist(user, firstname, lastname, email) VALUES ('$user','$firstname','$lastname','$email')"; 
+				if($conn->query($sql) === TRUE)
+					$message = "Contact Added!<br><br>";
+				else
+					$message = $conn->error;
+			}
+		}
+		else
```
Comment (ontraportgithub) [inline crmlogin/contactcreate.php]: Add brackets.

File: crmlogin/logincheck.php [hunk 2]
```diff
@@ -4,12 +4,20 @@
 	$password = $_POST['password'];
 	$enpass = hash('tiger192,3', $password);
 	echo $user." ".$enpass;
-	$sql = "SELECT username, password FROM users WHERE username='$user'";
-	$result = $conn->query($sql);
+	$sql = $conn->prepare("SELECT username, password FROM users WHERE username = ?");
```
Comment (ontraportgithub) [inline crmlogin/logincheck.php]: tick marks.

**Pull request merged**